projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -340,13 +339,9 @@ public function widget_non_instant( $args, $instance ) {
 			if ( ! Helper::are_filters_by_widget_disabled() && ! $this->should_display_sitewide_filters() ) {
 				$filters = array_filter( $filters, array( $this, 'is_for_current_widget' ) );
 			}
-
-			if ( ! empty( $filters ) ) {
-				$display_filters = true;
-			}
 		}
 
-		if ( ! $display_filters && empty( $instance['search_box_enabled'] ) && empty( $instance['user_sort_enabled'] ) ) {
+		if ( empty( $filters ) && empty( $instance['search_box_enabled'] ) && empty( $instance['user_sort_enabled'] ) ) {
```

Comment:
`$filters` will always exist, so you could just do like
```suggestion
		if ( ! $filters && empty( $instance['search_box_enabled'] ) && empty( $instance['user_sort_enabled'] ) ) {
```

Reply:
I'm not sure why I keep forgetting that empty arrays are falsy.